### DISCUSSION TITLE
Automated deployment to Maven Central

### DISCUSSION THREAD

**Discussion by Aloren** [aerospike/spring-data-aerospike#533] [Team Posts]
Hi @rbotzer 

For Maven Central publishing I would like to ask you to setup secrets in the spring-data-aerospike repository.
This can be done the following way: 
https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets

I will need OSSRH_USERNAME and OSSRH_TOKEN.
The values of the secrets can not be seen by any means neither in admin panel nor in github actions.

**Comment by rbotzer** [comment]
Sorry about the delay, was on vacation then and just saw this. Let me bring in @arrowplum for the task.

**Comment by arrowplum** [comment]
I'm sorry if I am being dense but what are the values of these tokens?

**Comment by Aloren** [comment]
I suppose these are the credentials of the user that is able to publish toMaven Central

**Comment by Aloren** [comment]
here is the guide how to publish to Maven Central via Github Actions: https://docs.github.com/en/actions/language-and-framework-guides/publishing-java-packages-with-maven#publishing-packages-to-the-maven-central-repository-and-github-packages

**Comment by rbotzer** [comment]
Sorry about the delay, we're in a crunch period.  We'll try to get to this in a few days.

**Comment by rbotzer** [comment]
I opened a Jira issue for it internally (OPS-4213), so I hope we can get to it soon.

**Comment by Aloren** [comment]
Okay, thanks. We are currently publishing a fork to our internal artifactory, so there's no hurry.

**Comment by Aloren** [comment]
Hi Ronen. Today I've had a talk with our devops, he said that even if we use group id `com.aerospike.spring` it will be published under `com.aerospike` folder, so it will require keys for `com.aerospike` group id...
